data/profiles/taskrunner.py

```diff
@@ -1,52 +1,62 @@
-#!/usr/bin/python
-
 # Copyright 2016, EMC, Inc.
 
 """
 Download and execute python scripts from the RackHD tasks api in a loop
 """
 
-import requests
+import urllib2
 import json
 import sys
 from time import sleep
 
-
 TASK_REQUEST_PERIOD = 5
-TASKS_URI = 'http://<%=server%>:<%=port%>/api/current/tasks/<%=identifier%>'
+TASKS_URI = 'http://<%=server%>:<%=port%>/api/2.0/tasks/<%=identifier%>'
 
 json_content_type = {"Content-Type": "application/json"}
 
 def download_task_data():
     """
     Download a python script from the tasks API
     """
-    print("Download Task", TASKS_URI)
-    session = requests.Session()
-    response = session.get(TASKS_URI, auth=("", ""))
-    _task_data = response.text
-    session.close()
+    _task_data = urllib2.urlopen(TASKS_URI).read()
```

Comment:
Same thing here.  We are going to need to work this into the logic to support both Brocade and Cisco or break into two different files.